pypeit/core/flux_calib.py

```diff
@@ -638,10 +639,12 @@ def sensfunc(wave, counts, counts_ivar, counts_mask, exptime, airmass, std_dict,
         ech_orders (int `numpy.ndarray`_):
             If passed the echelle orders will be added to the meta_table. ech_orders must be a numpy array of integers
             with the shape (norders,) giving the order numbers
-        mask_abs_lines (bool):
-            If True, mask stellar absorption lines before fitting sensitivity function. Default = True
+        mask_hydrogen_lines (bool):
+            If True, mask stellar hydrogen absorption lines before fitting sensitivity function. Default = True
+        mask_helium_lines (bool):
+            If True, mask stellar helium absorption lines before fitting sensitivity function. Default = False
         balm_mask_wid (float):
-            Parameter describing the width of the mask for or stellar absorption lines (i.e. mask_abs_lines=True). A region
+            Parameter describing the width of the mask for or stellar absorption lines (i.e. mask_hydrogen_lines=True). A region
```

Comment:
Does this also apply to the width of the helium line mask.

Reply:
Based on a few standard star spectra (and some background reading), I chose to empirically set the helium mask width to 0.5 the hydrogen width without introducing a separate user parameter.  If you think this should be fully adjustable, I can make it so.